packages/loadingIndicator/stories/loadingIndicator.stories.tsx

```diff
@@ -25,13 +33,13 @@ export const Template: Story = args => (
 
 export const _SectionLoadingIndicator = Template.bind({});
 
-export const _InlineLoadingIndicator = args => (
+export const _InlineLoadingIndicator = () => (
```

Comment:
can I ask why this change?

Reply:
It will display the color picker for the InlineLoadingIndicator, which you cannot set a color for this component. I'm not sure if it's best to break this out into its own story entirely or not. There are no props to pass into this component so it felt better to disable the controls by removing args as an option.